test/unit/subcommands/test_deploy_subcommand.py

```diff
@@ -34,56 +35,33 @@ def test_deploy_binaries_and_conf_deploys_both_conf_and_binary_for_remote_host(s
         self.assertTrue(mock_DeployTarget_instance.deploy_binary.called)
         self.assertTrue(mock_DeployTarget_instance.deploy_conf.called)
 
-    def test_deploy_binaries_and_conf_doesnt_deploy_conf_if_localhost_with_same_in_use_conf(self):
-        self.patch('os.path.expanduser').return_value = '/home'
-        mock_DeployTarget = self.patch('app.subcommands.deploy_subcommand.DeployTarget')
-        mock_DeployTarget_instance = mock_DeployTarget.return_value
-        deploy_subcommand = DeploySubcommand()
-        deploy_subcommand._deploy_binaries_and_conf(
-            'localhost', 'username', 'exec', '/path/to/exec', '/home/.clusterrunner/clusterrunner.conf')
-        self.assertFalse(mock_DeployTarget_instance.deploy_conf.called)
-
-    def test_deploy_binaries_and_conf_deploys_conf_if_localhost_with_diff_in_use_conf(self):
-        self.patch('os.path.expanduser').return_value = '/home'
-        mock_DeployTarget = self.patch('app.subcommands.deploy_subcommand.DeployTarget')
-        mock_DeployTarget_instance = mock_DeployTarget.return_value
-        deploy_subcommand = DeploySubcommand()
-        deploy_subcommand._deploy_binaries_and_conf(
-            'localhost',
-            'username',
-            'exec',
-            '/path/to/exec',
-            '/home/.clusterrunner/clusterrunner_prime.conf'
-        )
-        self.assertTrue(mock_DeployTarget_instance.deploy_conf.called)
-
```

Comment:
test_deploy_binaries_and_conf_doesnt_deploy_conf_if_localhost_with_same_in_use_conf
test_deploy_binaries_and_conf_deploys_conf_if_localhost_with_diff_in_use_conf
test_deploy_binaries_and_conf_deploys_binaries_if_localhost_and_different_executable_path_in_use
test_deploy_binaries_and_conf_doesnt_deploy_binaries_if_localhost_and_same_executable_path_in_use

are not very DRY. So I removed two tests and added asserts for deploy_conf for the remaining two. Right now it's testing (conf_in_use, exe_in_use) and (conf_not_in_use, exe_not_in_use) in two unit tests. I realize that it's missing the coverage of (conf_in_use, exe_not_in_use) and (conf_not_in_use, exe_in_use). Actually, I am going to try to consolidate these four unit tests into one.